brands/shop/car_repair.json

```diff
@@ -25,6 +25,16 @@
       "shop": "car_repair"
     }
   },
+  "shop/car_repair|BestDrive": {
+    "match": ["shop/car_repair|BestDrive"],
```

Comment:
Just a quick note - `match` is for less desirable taggings or spellings of `key/value|name`.  
You don't need to put the item's same key "shop/car_repair|BestDrive" into here, and this makes a warning.  I fixed the warning already, just letting you know for the future.